pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532</version>
+    <version>1.610</version>
```

Comment:
It's a runtime issue for Jenkins when using this plugin and running mvi hpi:run.  The moment you startup Jenkins with this plugin, you should be able to see the issue.